### DISCUSSION TITLE
Are there conversion tools that run on linux or windows ？

### DISCUSSION THREAD

**Discussion by lmx760581375** [Tencent/libpag#2349] [Q&A] [answered]
Are there conversion tools that run on linux or windows to convert images, video, audio to PAG or PAG back to these materials? I saw on the website that this tool only supports mac OS

**Comment by kevingpqi123** [comment, marked as answer]
The conversion tool is not a high-frequency use case, usually requiring only one or a few conversions. Due to limited resources, there are no plans to adapt it to other platforms. We suggest that you find a Mac machine to complete the conversions.

**Comment by CodeJhF** [comment]
Update: the conversion tool now supports Linux as well, and the macOS build is now a universal binary with native Apple Silicon support.

- macOS: https://pag.qq.com/file/PAGConvertor-mac.zip
- Linux (x86_64): https://pag.qq.com/file/PAGConvertor-linux.zip
- Linux (arm64): https://pag.qq.com/file/PAGConvertor-linux-arm64.zip

Each archive has a SHA-256 checksum file next to it, e.g. https://pag.qq.com/file/PAGConvertor-mac.zip.sha256

The download page has been updated: https://pag.io/docs/pag-convert-download.html
